src/app/components/Book/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import { styles } from './styles';
```

Comment:
this is okay but we usually export the styles as default so you can import it like this `import styles from './styles';`